Orange/data/util.py

```diff
@@ -28,3 +28,27 @@ def scale(values, min=0, max=1):
     if ptp == 0:
         return np.clip(values, min, max)
     return (-minval + values) / ptp * (max - min) + min
+
+
+class ComputeValueWCommon:
```

Comment:
I don't know. Will be used for other transformations as part of this PR? Utils are tools, like screwdrivers and wrenches. This doesn't look like a tool, rather a very specific piece of plug I, unrelatedly, happen to not appreciate at all. :smiley: